bootstrapvz/common/tasks/ssh.py

```diff
@@ -109,4 +110,8 @@ def run(cls, info):
         public = [path + '.pub' for path in private]
 
         from ..tools import log_check_call
-        log_check_call(['shred', '--remove'] + private + public)
+        for ssh_host_key in private + public:
+            if os.path.isfile(ssh_host_key):
```

Comment:
Yeah DSA keys are not in Stretch but it is not set in stone that somebody will not use them and to be honest we should remove all keys which are generated during bootstrapping. So I'd say in this case removing all of them silently is not a big issue.

You're right it could be clearer is `host_key` ok with you?